package_bag/helpers.py

```diff
@@ -0,0 +1,17 @@
+
+
+def expected_file_name(filename):
+    """Returns true if a filename matches an expected bag name
+
+    Expected bag names are 32 characters before a file extension, and have a file extension of .tar or .tar.gz
+
+    Args:
+        filename (string): a filename (not including full path)
+
+    Returns:
+        boolean: True if filename matches, false otherwise"""
+    filename_split = filename.split(".")
```

Comment:
Splitext does not handle .tar.gz files as simply. E.g., with `4b4334fba43a4cf4940f6c8e6d892f60.tar.gz`,

Splitext: `('4b4334fba43a4cf4940f6c8e6d892f60.tar', '.gz')`
Split("."): `['4b4334fba43a4cf4940f6c8e6d892f60', '.tar', '.gz']`

Since I want to check the length of the filename before the extension in addition to checking the extension, I went with `split`.

Reply:
OK, gotcha. That makes sense.